incubator/binding-openapi/src/main/java/io/aklivity/zilla/runtime/binding/openapi/internal/config/OpenapiServerCompositeBinding.java

```diff
@@ -56,40 +55,39 @@
 import io.aklivity.zilla.runtime.engine.config.GuardedConfigBuilder;
 import io.aklivity.zilla.runtime.engine.config.MetricRefConfig;
 import io.aklivity.zilla.runtime.engine.config.ModelConfig;
+import io.aklivity.zilla.runtime.engine.config.NamespaceConfig;
 import io.aklivity.zilla.runtime.engine.config.NamespaceConfigBuilder;
 import io.aklivity.zilla.runtime.engine.config.RouteConfigBuilder;
 import io.aklivity.zilla.runtime.model.json.config.JsonModelConfig;
 
-public final class OpenapiServerCompositeBindingAdapter extends OpenapiCompositeBindingAdapter
+public final class OpenapiServerCompositeBinding extends OpenapiCompositeBinding
```

Comment:
I think the term `binding` is overloaded here, and a bit confusing as it doesn't mean the same thing as binding means elsewhere.

Previously composite binding adapter was being used during parse, but now we are just generating a namespace.

Suggest we call these `OpenapiServerNamespaceGenerator` and `OpenapiNamespaceGenerator`, then call the method `generate` (a verb) instead of `composite` (a noun).